scripts/upgrade-version.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
is this the correct directory for this, would assume this is either in `bin/` or root